client/src/components/Explanation/Explanation.js

```diff
@@ -0,0 +1,73 @@
+import styled from 'styled-components/macro'
+import CardWrapper from '../../styled/CardWrapper'
+import ComponentsWrapper from '../../styled/ComponentsWrapper'
+import H2 from '../../styled/H2'
+import H3 from '../../styled/H3'
+import ParagraphCenter from '../../styled/ParagraphCenter'
+
+const Explanation = () => {
+  return (
+    <ComponentsWrapper>
+      <CardWrapper>
+        <H2>Explanation</H2>
+        <ParagraphCenter>
+          Be objective gives you the opportunity to rate two movies against each
+          other. You decide which movie you like more.
+        </ParagraphCenter>
+        <ParagraphCenter>
+          Once you made your decision your vote will give points to the winner
+          which are subtracted from the loser. For this we use the elo-rating
+          system.
+        </ParagraphCenter>
+      </CardWrapper>
+      <CardWrapper>
+        <H3>Voting</H3>
+        <ParagraphCenter>The voting looks like this:</ParagraphCenter>
+        <ImgVoting
+          src="/explanation/voting.png"
+          width="348"
+          height="582"
+          alt="VotingArea"
+        />
+        <ParagraphCenter>
+          On the top and the bottom you find the movies that are competing with
+          each other. In the middle you find the buttons that you can use.
+        </ParagraphCenter>
+        <H3>Unwatched Movies</H3>
+        <ParagraphCenter>
+          The red buttons on the left side are the 'unwatched' buttons. If you
+          haven't watched one or both movies you can click on the corresponding
+          button. The app will then save this and not show the movie again.
```

Comment:
Maybe "won't" instead of "not" in "The app will then save this and not show the movie again." 🤔

Reply:
Good one again, thx!